internal/server/main_test.go

```diff
@@ -0,0 +1,282 @@
+package server
+
+import (
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_updateGauge(t *testing.T) {
```

Comment:
Underscore в принципе позволителен в названиех, но вообще принято просто camel case использовать
```suggestion
func TestUpdateGauge(t *testing.T) {
```
На проектах тебя попросят исправить, поэтому давай привыкать придерживаться мнению комьюнити и будем использовать camel case)